### PR TITLE
fix: shop list view img size

### PR DESCRIPTION
**Pull request by Weebian**
fix: shop list view img size

### CODE REVIEW

**Review by DwangDistracted — APPROVED** [review]

**Comment by dershao** [comment]
image

Result after I add the css

File: shopify-client/src/views/ShopList.vue [hunk 1]
```diff
@@ -242,4 +242,9 @@ input{
   margin: 5px;
   vertical-align: top;
 }
+
+.card-img-top{
```
Comment (Souheil-Yazji) [inline shopify-client/src/views/ShopList.vue:246]: I suggest going to line 63 and replacing the img tag with a b-img tag and using the "fluid" prop, see https://bootstrap-vue.org/docs/components/image#responsive-images

**Review by Souheil-Yazji — CHANGES_REQUESTED** [review]

File: shopify-client/src/views/ShopList.vue (hunk 1 above)
Comment (Souheil-Yazji) [reply shopify-client/src/views/ShopList.vue:246]: PS: Remember that we are using BootstrapVue to make our lives easier, take advantage of it! You don't have to go through the pain of writing CSS when Bootstrap provides you with the resources to simplify styling.

Comment (Weebian) [reply shopify-client/src/views/ShopList.vue:246]: i did the changes you asked, but it does not resolve the issue at hand
I will still need some css

**Review by dershao — APPROVED** [review]

**Review by Souheil-Yazji — APPROVED** [review]
Lgtm :)

**Pull request merged**